CMakeLists.txt

```diff
@@ -35,6 +35,9 @@ cmake_minimum_required (VERSION 3.0)
 
 project ("rttr" LANGUAGES CXX)
 
+set(CMAKE_CXX_EXTENSIONS OFF)
```

Comment:
Why do you disable the CXX extensions?
This is the default behavior in CMake, I would stick to it.
Otherwise you can tell me a good reason why not.